docs/public/specifications/tests/Connectivity-TP/connectivity04.md

```diff
@@ -0,0 +1,252 @@
+# CONNECTIVITY04: IP Prefix Diversity
```

Comment:
@tgreenx, maybe "network prefix" is a better term. See https://www.ripe.net/about-us/press-centre/understanding-ip-addressing and https://en.wikipedia.org/wiki/IP_address#Subnetworks

Such a change would have consequences in many places.

Reply:
I don’t know. I don’t have any issue with either “IP prefix” or “network prefix”. However, if it turns out that “IP prefix” is a bad term, I suggest opening a separate issue.